forms/userEdit.html

```diff
@@ -0,0 +1,316 @@
+                                <div class="modal-body-user-role-role">
+                                  <span class="role role-manager margin-5px">M</span>
+                                </div>
+                                <div class="modal-body-user-role-description">
+                                  <h2>Manager</h2>
+                                  <p>
+                                    <ul>
+                                      <li>
+                                        <span class="bold">Can</span> <span class="underlined">add</span>, <span class="underlined">  change</  span> or <span class="underlined">delete</span> users
+                                      </li>
+                                      <li>
+                                        <span class="bold">Can't</span> <span class="underlined">read</span>, <span class="underlined"> write</span> or <span class="underlined">change</span> users
+                                      </li>
+                                    </ul>
+                                  </p>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                          <div class="modal-footer">
+                            <button id="modal-close-button" class="button">CLOSE</button>
+                          </div>
+                        </div>
+                      </div>
+                    <!-- Modal stops here -->
+                  </div>
+                </div>
+              </div>
+              <div class="question-save">
+                <div class="question-main-filler-left">
+                </div>
+                  <a href="userTable.html">
+                    <button class="button">SAVE</button>
+                  </a>
+              </div>
+            <!-- hardcoded questions stop here -->
+          </div>
+        <!--
+          {% endblock mainContent %}
+        -->
+        <div class="side-filler">
+        </div>
+      </main>
+      <footer>
+        <div class="side-filler">
+        </div>
+        <div id="footer-content">
+          <div id="footer-left" class="padding-10px">
+            Documentation<i class="material-icons icon-large">local_library</i>
+          </div>
+          <div id="footer-right" class="padding-10px">
+            <i class="material-icons icon-large">headset_mic</i>Support
+          </div>
+        </div>
+        <div class="side-filler">
+        </div>
+      </footer>
+    </div>
+  </body>
+  <script>
+    // Get the modal
+    var modal = document.getElementById('about-roles-modal');
+    // Get the button that opens the modal
+    var buttonAboutRoles = document.getElementById("about-roles-button");
+    // Get the <span> element that closes the modal
+    var buttonModalClose = document.getElementById("modal-close-button");
+    // When the user clicks the button, open the modal
```

Comment:
Bij klikken liever een class toevoegen versus weghalen.
Bijvoorbeeld de class: not-shown

in css 
.not-shown {
  display: none;
} 
kan zoeken welke manier van hiden je wil gebruiken.